example/glue/ObjectModel.hpp

```diff
@@ -149,7 +150,7 @@ class GC_ObjectModel
 	MMINLINE bool
 	isDeadObject(void *objectPtr)
 	{
-		return J9_GC_OBJ_HEAP_HOLE == (OMR_OBJECT_FLAGS(objectPtr) & J9_GC_OBJ_HEAP_HOLE_MASK);
+		return J9_GC_OBJ_HEAP_HOLE == (OMR_OBJECT_FLAGS(objectPtr) & J9_GC_OBJ_HEAP_HOLE);
```

Comment:
Without this change heap iterator returns single slot holes (dead object) as live objects.